src/charactersheet/viewmodels/dm/root/index.js

```diff
@@ -130,29 +126,16 @@ export function DMRootViewModel() {
     self.load = () => {
         self.activeTab(TabFragmentManager.activeTab());
 
-        Notifications.party.joined.add(self._updateCurrentNode);
-        Notifications.party.left.add(self._removeCurrentNode);
-        Notifications.xmpp.disconnected.add(self._removeCurrentNode);
-
         HotkeysService.registerHotkey('1', self.activateOverviewTab);
         HotkeysService.registerHotkey('2', self.activateEncounterTab);
         HotkeysService.registerHotkey('3', self.activateDmScreenTab);
         HotkeysService.registerHotkey('4', self.activateNotesTab);
```

Comment:
do these hot keys match the order in the html?

Reply:
<img width="518" alt="Screen Shot 2021-12-08 at 4 06 25 PM" src="https://user-images.githubusercontent.com/3310280/145310785-453c0402-923b-4319-b4ff-a0cdae08c1b6.png">